src/main/res/values/strings.xml

```diff
@@ -941,10 +942,10 @@
     </string-array>
     <string-array translatable="false" name="tagfilter_type_values">
         <item>*</item>
-		<item>node</item>
```

Comment:
This shouldn't be changed as it is an internally used string value (as in general anything with translatable="false" set).

Reply:
Appeared strange that it wasn't the same as the otherwise duplicated list above it, reverted.